starter-code/index.html

```diff
@@ -0,0 +1,261 @@
+              <p>Quisque sit amet sagittis justo. Suspendisse ornare, felis feugiat suscipit pharetra, lorem est viverra odio; tristique tristique diam tellus id tellus. In consectetur aliquet tristique. Curabitur imperdiet leo in nisl consectetur mattis. Quisque bibendum, mauris vitae suscipit ornare; est nibh elementum mauris, eu malesuada nunc magna at justo. Proin blandit id tellus non pellentesque. Mauris […]</p>
+              <button type="button" class="btn btn-outline-light">Read More</button>
+            </div>
+          </div>
+          <div class="carousel-item">
+            <img src="./img/slider-2.jpg" class="d-block w-100" alt="olives">
+            <div class="carousel-caption d-none d-md-block">
+              <h5>Third slide label</h5>
+              <p>Nam luctus velit ultricies vestibulum eleifend. Proin imperdiet varius rutrum. Curabitur placerat lacus eu ligula pellentesque euismod? Aliquam volutpat augue id nibh auctor, at varius dui semper? Suspendisse non pellentesque sapien, et sagittis erat. Quisque et dignissim elit. Sed sit amet sapien ac quam vestibulum suscipit ac id diam. Morbi accumsan est vitae tellus vulputate […]
+            </p>
+              <button type="button" class="btn btn-outline-light">Read More</button>
+            </div>
+          </div>
+          <div class="carousel-item">
+            <img src="./img/slider-1.jpg" class="d-block w-100" alt="kitten-and-puppy-sleeping">
+            <div class="carousel-caption d-none d-md-block ">
+              <h5>ETIAM MAURIS TORTOR, PHARETRA QUIS LOBORTIS IN, PHARETRA IN DIAM</h5>
+              <p>Duis tempus leo vitae ipsum viverra, blandit condimentum sapien porttitor. Duis porttitor sed metus eget mollis. Curabitur bibendum imperdiet tortor, ut pulvinar purus elementum nec. Suspendisse at erat luctus; hendrerit sapien sed, consectetur erat. Maecenas dignissim suscipit orci at molestie. Aenean fringilla dolor vitae lacus lacinia eleifend. Suspendisse et libero nunc. Etiam mauris tortor, pharetra […]</p>
+              <button type="button" class="btn btn-outline-light">Read More</button>
+            </div>
+          </div>
+        </div>
+       
+        </div>
+        <a class="carousel-control-prev" href="#carouselExampleCaptions" role="button" data-slide="prev">
+          <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+          <span class="sr-only">Previous</span>
+        </a>
+        <a class="carousel-control-next" href="#carouselExampleCaptions" role="button" data-slide="next">
+          <span class="carousel-control-next-icon" aria-hidden="true"></span>
+          <span class="sr-only">Next</span>
+        </a>
+      </div>    
+
+   </div>
+   <!-- container 3  --> 
+   <div class="container-fluid container3">
+       <div class="row">
+            <div class="col-sm-4 homewidget container3-div">
+                <!--<span><i class="glyphicon glyphicon-glass"></i></i>  </span> Not working...-->
```

Comment:
https://www.w3schools.com/icons/bootstrap_icons_glyphicons.asp
Note: Glyphicons are not supported in Bootstrap 4.

FYI